### ISSUE TITLE
[Browser] The object reference was not set to an object instance.

### ISSUE THREAD

**Issue by msebastian100** [JonasNilson/idle_master_extended#285]
```
12.04.2022 10:38:41   
System.NullReferenceException: Der Objektverweis wurde nicht auf eine Objektinstanz festgelegt.
   bei IdleMasterExtended.frmBrowser.tmrCheck_Tick(Object sender, EventArgs e)
   bei System.Windows.Forms.Timer.OnTick(EventArgs e)
   bei System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

**Comment by JonasNilson** [comment]
This is likely happening when loading the login page in the browser window and the timer starts checking before all the objects have been loaded.

Is this an error that is stopping you from using the application?

**Comment by JonasNilson** [comment]
This issue is resolved in the v1.8 release: https://github.com/JonasNilson/idle_master_extended/releases

**Comment by JonasNilson** [comment]
The latest release removes the browser window: https://github.com/JonasNilson/idle_master_extended/releases/tag/v1.8.0

Closing.
